### ISSUE TITLE
Contact page is confusing

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#217]
It looks like a blog page. User also may not know the term &quot;CONTACT AND ENGAGE&quot; (see &quot;Write the page in plain language&quot; issue)

**Comment by ErieMeyer** [comment]
Hi all!

Taking a crack at simplifying/making it more plain language. See suggestions below:

![image](https://f.cloud.github.com/assets/1626026/1737479/cdefe686-6373-11e3-8109-bc7401d961ba.png)
1. Replace with "Contact" both to match nav and to make sure we're using copy that's meaningful (engage is little fuzzy)
2. Replace this content with "How can we help?" to reduce length, complexity, and duplication.
3. Replace "Ask the open data community" with "Open Data Stack Exchange" both to match the structure of the twitter column and to simplify. Replace the blurb below with "Ask the community."
4. Replace the blurb below "Twitter" with "Follow us at @usdatagov"
5. Delete "Recent Tweets" header, since styling should make clear they're the recent tweets. Delete old logo (will be duplicative once we switch everything to the same new logo). 
6. Delete sub-copy here to reduce distraction.

![image](https://f.cloud.github.com/assets/1626026/1737448/4d56dd9a-6373-11e3-9752-f031a746be69.png)

7 Whoops! Don't actually need to change anything on this.

8 Delete this (too many headers for this portion and styling makes clear it's a contact form)

9 Delete this. Messages are a) not private (could be FOIAd), and b) styling/behavior of forms makes clear this is going directly to someone (rather than being posted)

10 Delete "Address" since "Email" is self explanatory.

11 Switch this copy to read "How can we help?"

12 Delete this copy. Form function is self evident, the messages are not "private," and we should only use a .gov email here instead of the email address posted (also not sure this email address is monitored? Send a test message today).

13 Delete "with data.gov" (It's self explanatory without the extra copy)

Hope this helps!

**Comment by kvuppala** [comment]
Trying to get most of these done in tonight's push. Email address is currently being linked to JIRA systems which is monitored, if any .gov email exists we can switch that.

**Comment by ErieMeyer** [comment]
Thanks!

A few more suggested changes:

![image](https://f.cloud.github.com/assets/1626026/1756905/9b6877e2-6679-11e3-8b4c-126ca8b2fe10.png)
- Delete the large "Twitter: usdatagov" and icon, so that the twitter area matches the cleanliness of the stack exchange area.
- Switch "Follow us @usdatagov" under the twitter icon to "Ask us" (matches stack exchange phrasing).
- Drop "From tweetdeck" and "ReplyRetweetFavorite" under the tweet
- Is there any way to style the tweets so they're easier to read? The stack exchange formatting makes clear that each post is a question, when it was posted, etc. Would be great to borrow that styling here.

![image](https://f.cloud.github.com/assets/1626026/1756948/4831fd72-667a-11e3-86e6-a7e49e4bd6e0.png)
- Make "data" lower case here.

Thanks!

**Comment by philipashlock** [comment]
@kvuppala I'm working on completely restyling the page to more closely match the mockup and should have an update soon.

**Comment by philipashlock** [comment]
These changes have all been implemented.
